pkg/vfs/internal.go

```diff
@@ -243,19 +243,19 @@ func (v *VFS) handleInternalMsg(ctx meta.Context, cmd uint32, r *utils.Buffer, d
 			return
 		}
 		var w = bytes.NewBuffer(nil)
-		fmt.Fprintf(w, " inode: %d\n", inode)
-		fmt.Fprintf(w, " files:\t%d\n", summary.Files)
-		fmt.Fprintf(w, " dirs:\t%d\n", summary.Dirs)
-		fmt.Fprintf(w, " length:\t%d\n", summary.Length)
```

Comment:
We may want to know the exact number, so this should be controlled by a flag.

Reply:
Both formats will be printed, like:
```
/Users/sandy/jfs :
  inode: 1
  files: 9
   dirs: 2
 length: 105.00 MiB (110100528 Bytes)
   size: 105.04 MiB (110141440 Bytes)
   path: /
```